data/dataset/snowflake_example_test_dataset.yml

```diff
@@ -0,0 +1,214 @@
+dataset:
+  - fides_key: snowflake_example_test_dataset
+    name: Snowflake Example Test Dataset
+    description: Example of a Snowflake dataset containing a variety of related tables like customers, products, addresses, etc.
+    collections:
+      - name: address
+        fields:
+          - name: city
+            data_categories: [user.provided.identifiable.contact.city]
+          - name: house
+            data_categories: [user.provided.identifiable.contact.street]
+          - name: id
+            data_categories: [system.operations]
+            fidesops_meta:
+              primary_key: True
+          - name: state
+            data_categories: [user.provided.identifiable.contact.state]
+          - name: street
+            data_categories: [user.provided.identifiable.contact.street]
+          - name: zip
+            data_categories: [user.provided.identifiable.contact.postal_code]
+
+      - name: customer
+        fields:
+          - name: address_id
+            data_categories: [system.operations]
+            fidesops_meta:
+              references:
+                - dataset: snowflake_example_test_dataset
+                  field: address.id
+                  direction: to
+          - name: created
+            data_categories: [system.operations]
+          - name: email
+            data_categories: [user.provided.identifiable.contact.email]
+            fidesops_meta:
+              identity: email
```

Comment:
heads up that after https://github.com/ethyca/fidesops/pull/108 is merged, we'll require `data_type` on applicable fields in erasure tests

Reply:
Thanks!